docs/README.md

```diff
@@ -0,0 +1,166 @@
+
+   ```bash
+   git submodule update --init --recursive
+   ```
+
+### Executing Terraform Scripts
+
+Before running the Terraform scripts, ensure your environment is correctly configured.
+
+1. **AWS Region and Profiles**
+
+   Set the `AWS_REGION` and `AWS_PROFILE` environment variables:
+
+   ```bash
+   export AWS_REGION=<your-region>
+   export AWS_PROFILE=<workspace>
+   ```
+
+   - Replace `<your-region>` with your desired AWS region (e.g., `us-east-1`).
+   - Replace `<workspace>` with either `dev` or `prod`, matching the Terraform workspace you intend
+     to use.
+
+   **Note:** You must have AWS CLI profiles named `dev` and `prod` configured in your
+   `~/.aws/credentials` or `~/.aws/config` files. Each profile should correspond to the AWS account
+   for the respective environment.
+
+2. **Terraform Workspaces**
+
+   The project utilizes two Terraform workspaces: `dev` and `prod`, corresponding to development and
+   production environments. Switch between workspaces using:
+
+   ```bash
+   terraform workspace select dev   # Switch to the development environment
+   terraform workspace select prod  # Switch to the production environment
+   ```
+
+   See the [Documentaiton](#documentation) for more information on which workspace to use for each
+   configuration.
+
+3. **Running Terraform**
+
+   Initialize the Terraform working directory and apply the configuration:
+
+   ```bash
+   terraform init
+   terraform apply
+   ```
+
+   Review the execution plan carefully before confirming the changes.
+
+### Running Locally with Docker Compose
+
+This repository allows running the Pillarbox Monitoring solution locally. To do so first start
+services using Docker Compose:
+
+```bash
+docker-compose up --build
+```
+
+This command will build and run the following services:
+
+- **pillarbox-monitoring-dispatch**: Receives events via HTTP POST and broadcasts them via SSE.
+- **pillarbox-monitoring-transfer**: Listens to SSE events and stores them in OpenSearch.
+- **OpenSearch**: Stores the events for analysis.
+- **Grafana**: Visualizes the data stored in OpenSearch.
+
+**Sending Events**: You can send events to the system using the following command:
+
+ ```bash
+ curl -X POST http://localhost:8080/api/events \
+      -H 'Content-Type: application/json' \
+      -d "{\"msg\": \"data\", \"timestamp\": \"$(date +%s)\"}"
+ ```
+
+**Accessing Grafana**: Grafana is accessible at [http://localhost:3000](http://localhost:3000).
+
+## Documentation
+
+The project is split into four main Terraform configurations:
+
+- [01-terraform-backend][terraform-backend]: This configuration sets up the Terraform backend and is
+  **only applicable in the prod environment**. The backend is used to store Terraform state
+  remotely, ensuring that multiple users and systems can work on infrastructure collaboratively.
+- [10-pillarbox-monitoring-route-53][route-53]: This handles the creation of the Route 53 DNS setup
```

Comment:
[Route 66](https://www.youtube.com/watch?v=AqhQSfFtOVE) would have been nicer. You're welcome.

Reply:
You can contact AWS: https://aws.amazon.com/contact-us/ 😂